build.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     testImplementation("org.junit.jupiter", "junit-jupiter", "5.10.2")
     testImplementation("org.assertj", "assertj-core", "3.25.3")
     testImplementation("io.kotest", "kotest-runner-junit5", "5.8.0")
+    testImplementation("io.kotest", "kotest-framework-datatest", "5.7.2")
```

Comment:
Actually, it seems you weren't aware of this dependency. May I ask why you decided to add it?

Reply:
Oh I was using data tests in all other missions so I added beforehand. I did not need it for step1 but for next steps, I will be using them!